.php_cs

```diff
@@ -22,25 +22,31 @@ return PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
         '@Symfony' => true,
-        'binary_operator_spaces' => ['align_double_arrow' => true, 'align_equals' => true],
-        'array_syntax' => array('syntax' => 'short'),
-        'blank_line_after_namespace' => true,
-        'header_comment' => array('header' => $header),
-        'lowercase_constants' => true,
-        'lowercase_keywords' => true,
-        'method_separation' => true,
-        'no_closing_tag' => true,
-        'no_extra_consecutive_blank_lines' => [
+        'array_syntax' => [
+            'syntax' => 'short'
+        ],
+        'binary_operator_spaces' => [
+            'align_double_arrow' => true,
+            'align_equals'       => true
+        ],
+        'class_attributes_separation' => [
+            'elements' => [ "const", "method", "property" ]
+        ],
+        'header_comment' => [
+            'header' => $header
+        ],
+        'linebreak_after_opening_tag' => true,
+        'no_extra_blank_lines' => [
             'tokens' => ['break', 'continue', 'extra', 'return', 'throw', 'use', 'parenthesis_brace_block', 'square_brace_block', 'curly_brace_block']
         ],
         'no_short_echo_tag' => true,
-        'no_useless_else' => true,
+        'no_useless_else'   => true,
         'no_useless_return' => true,
-        'ordered_imports' => true,
-        'phpdoc_add_missing_param_annotation' => true,
+        'ordered_imports'   => true,
+        'phpdoc_add_missing_param_annotation' => [
+            'only_untyped' => false
```

Comment:
i would set it to true
(phpdoc is not as usefull as before if we use type params, feels duplicate info)

Reply:
Ok, I'll delete this configuration parameter, `true` is the default value.